### ISSUE TITLE
Deprecated Functionality: Unparenthesized `a ? b : c ?: d` is deprecated

### ISSUE THREAD

**Issue by rafaelola** [MyParcelCOM/magento2-plugin#8]
Deprecated Functionality: Unparenthesized `a ? b : c ?: d` is deprecated. Use either `(a ? b : c) ?: d` or `a ? b : (c ?: d)` in /vendor/myparcelcom/magento/Model/Sales/MyParcelOrderCollection.php on line 169

**Comment by rafaelola** [comment]
This changed fixed it:
```
$items[] = (new ShipmentItem())
                    ->setSku($product->getSku())
                    ->setDescription($product->getName())
                    ->setImageUrl($imageUrl)
                    ->setItemValue($this->amountInCents($orderItem->getPrice()))
                    ->setCurrency($order->getOrderCurrencyCode())
                    ->setQuantity((int) $orderItem->getQtyOrdered())
                    ->setHsCode(!empty($product->getData('hs_code'))
                        ? $product->getData('hs_code')
                        : $myparcelExportSetting['default_hs_code'] ?: null
                    )
                    ->setItemWeight($this->weightInGrams($orderItem->getWeight()))
                    ->setOriginCountryCode(!empty($product->getData('country_of_manufacture'))
                        ? $product->getData('country_of_manufacture')
                        : $myparcelExportSetting['default_origin_country_code'] ?: null
                    )
                    ->setVatPercentage((int) $orderItem->getTaxPercent());
```

**Comment by M4tini** [comment]
Thanks for your report, this has been fixed in release [v1.2.1](https://github.com/MyParcelCOM/magento2-plugin/releases/tag/v1.2.1).
